tests/Resources/ti.ui.view.test.js

```diff
@@ -1085,6 +1085,39 @@ describe('Titanium.UI.View', function () {
 			win.add(outerView);
 			win.open();
 		});
+
+		it.ios('set property post layout', finish => {
+			win = Ti.UI.createWindow({ backgroundColor: 'blue' });
+			const outerView = Ti.UI.createView({
+				width: '90px',
+				height: '90px',
+				backgroundColor: 'green'
+			});
+			const view = Ti.UI.createView({
+				width: '60px',
+				height: '60px',
+				backgroundColor: 'yellow'
+			});
+
+			win.addEventListener('postlayout', function postlayout() {
+				win.removeEventListener('postlayout', postlayout); // only run once
+				try {
+					view.borderRadius = [ '12px', 12 ];
+					should(view.borderRadius).be.an.Array();
+					should(view.borderRadius.length).eql(2);
+					should(view.borderRadius).eql([ '12px', 12 ]);
+					// should be the exact same as above
+					should(outerView).matchImage(`snapshots/borderRadius12px_12_${density}x.png`);
```

Comment:
<!--
  1 failure:  tests/Resources/t...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.view.test.js;
  Line: 1110;
-->
- :no_entry_sign: tests/Resources/ti.ui.view.test.js line 1110 – 'density' is not defined. (no-undef)